awscli/customizations/s3/subcommands.py

```diff
@@ -369,6 +369,15 @@
                         'output is suppressed.')}
 
 
+NO_PROGRESS = {'name': 'no-progress',
+               'action': 'store_false',
+               'dest': 'progress',
```

Comment:
Not sure it is needed but all of the parameters like this have a `'default': True` in the dictionary. Not sure if it is needed. Do not think it is. But it would not hurt to be consistent.

Reply:
The other two flags like this don't have a default either, I think it's fine to have no default.

Reply:
I was referring to all of the boolean flags that have `store_false` as the action. I just checked the docs and did it on the command line and its fine. Putting the `default` kwarg whose value is `True` is redundant.